app-web/registry/developer-tools.json

```diff
@@ -29,6 +29,17 @@
           ]
         }
       },
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/bcgov/devhub-resources",
```

Comment:
just curious, why do you need the full url and the owner+repo ?

Reply:
i think this was an artifact of the very first version of devhub. I don't think it actually has any relevance anymore...